phpBB/language/en/cli.php

```diff
@@ -110,8 +110,10 @@
 
 	'CLI_EXTENSION_DISABLE_FAILURE'		=> 'Could not disable extension %s',
 	'CLI_EXTENSION_DISABLE_SUCCESS'		=> 'Successfully disabled extension %s',
+	'CLI_EXTENSION_ALREADY_DISABLED'	=> 'Extension %s was already disabled',
```

Comment:
This message should probably be "Extension %s is not enabled".  An extension might be neither enabled nor disabled (available, or even, non-existent)...